spec/comment_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+
+RSpec.describe Comment, type: :model do

```

Comment:
- [ ] All methods in your Comment model should be tested as mentioned in the project requirements. Please write a test for the method `update_counter_posts`